src/applications/claims-status/components/ClaimsListItemV3.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { getClaimType } from '../utils/helpers';
+
+const statusMap = {
+  CLAIM_RECEIVED: 'Step 1 of 5: Claim received',
+  INITIAL_REVIEW: 'Step 2 of 5: Initial review',
+  EVIDENCE_GATHERING_REVIEW_DECISION:
+    'Step 3 of 5: Evidence gathering, review, and decision',
+  PREPARATION_FOR_NOTIFICATION: 'Step 4 of 5: Preparation for notification',
+  COMPLETE: 'Step 5 of 5: Closed',
+};
+
+function getStatusDescription(status) {
+  return statusMap[status];
+}
+
+const formatDate = date => moment(date).format('MMMM D, YYYY');
+
+const getTitle = claim => {
+  return `Claim for ${getClaimType(claim)}`;
+};
+
+const getLastUpdated = claim => {
+  const updatedOn = formatDate(
+    claim.attributes.claimPhaseDates?.phaseChangeDate,
+  );
+
+  return `Last updated: ${updatedOn}`;
+};
+
+const isClaimComplete = claim => {
+  const { decisionLetterSent, status } = claim.attributes;
+
+  return decisionLetterSent || status === 'COMPLETE';
```

Comment:
One thing to keep in mind here is this doesn't just return true when the claim is closed, so we may want to reconsider the logic